src/tests/Loader/classloader/InlineArray/InlineArrayValid.cs

```diff
@@ -0,0 +1,367 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
The GC manipulation triggers in here mean we should run this locally under GC stress to make sure we don't have any asserts firing. Can you try that quickly please? It also might make be prudent to run the JIT stress run legs.

Reply:
right, once we done with making changes I will run some GC/JIT stress.

Reply:
I've run tests locally with JIT GC hole stress settings (DOTNET_GCStress=0x3, higher setting takes very long time). 
There were no failures.